src/components/Modals/Metadata/Metadata.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Box, Button, Image, Text } from 'grommet'
+import { Close } from 'grommet-icons'
+import styled from 'styled-components'
+import data from './mockData'
+import Satellite from 'images/satellite_map.png'
+import ConditionalLink from './components/ConditionalLink'
+
+const Uppercase = styled(Text)`
+  text-transform: uppercase;
+`
+
+export default function Metadata() {
+  return (
+    <Box
+      border={{ color: 'kelp' }}
+      elevation='small'
+      gap='xsmall'
+      height='32rem'
+      pad='medium'
+      width='27rem'
+    >
+      <Box direction='row' justify='between'>
+        <Text color='kelp'>Subject Metadata</Text>
+        <Button
+          gap='xsmall'
+          icon={<Close color='black' size='small' />}
+          label={<Uppercase color='kelp' size='small'>Close</Uppercase>}
+          onClick={() => console.log('Close the modal')}
+          plain
+          reverse
```

Comment:
We have a `PlainButton` in the react-components lib that has these styles, built on top of the Grommet one, so should be able to use `reverse` too: https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/plainbutton--with-icon